css/style.css

```diff
@@ -4,16 +4,163 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0px;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
 
+.links {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    
+    }
 
+a {
+    margin: 20px;
+    text-decoration: none;
+    color:grey;
+}
+
+
+a:hover {
+    color: darkorange;
+}
+
+.logo {
+  display: inline-flex;
+  height: auto;
```

Comment:
It's not wrong to have this but for your information, if you don't set the height, it will be auto